contrib/src/main/java/gov/nasa/jpl/aerie/contrib/streamline/modeling/Registrar.java

```diff
@@ -64,17 +59,13 @@ public enum ErrorBehavior {
     Throw
   }
 
-  public Registrar(final gov.nasa.jpl.aerie.merlin.framework.Registrar baseRegistrar, final ErrorBehavior errorBehavior) {
+  public Registrar(final gov.nasa.jpl.aerie.merlin.framework.Registrar baseRegistrar, final Instant planStart, final ErrorBehavior errorBehavior) {
```

Comment:
If we can avoid changing the signature of the Registrar constructor, I think this PR can be backwards compatible with code that already uses the streamline framework